### PR TITLE
Display battery usage on bluetooth radio screen

### LINKED ISSUE #847: 🚀 [Feature Request]: Add battery level to the bluetooth radio connect view
### OS

iOS

### Description

It would be nice to surface the battery level more conveniently, since it's such a common thing to look at.

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by gabek**
## What changed?

This change adds the battery level to the bluetooth radio status screen. The discussion came up in #837. Closes #847

## Why did it change?

I felt the battery status should be surfaced more conveniently, since it is a common value to be looking at. 

## Checklist

- [x] My code adheres to the project's coding and style guidelines.
- [x] I have conducted a self-review of my code.
- [ ] I have commented my code, particularly in complex areas.
- [ ] I have made corresponding changes to the documentation.
- [x] I have tested the change to ensure that it works as intended.

### CODE REVIEW

File: Meshtastic/Helpers/BatteryHelper.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  BatteryHelper.swift
+//  Meshtastic
+//
+//  Created by Gabe Kangas on 8/6/24.
+//
+
+import Foundation
+import MeshtasticProtobufs
+
+class BatteryHelper {
```
Comment (72A12F4E) [inline Meshtastic/Helpers/BatteryHelper.swift]: this can be an `enum`, since we won't ever need to instantiate an instance of `BatteryHelper`

**Comment by 72A12F4E** [comment]
LGTM overall

**Comment by garthvh** [comment]
I would rather use the existing battery level indicator that is on the node list, under the circle.

**Comment by garthvh** [comment]
Battery indicator from node details has been added to the bluetooth screen

**Pull request closed without merging**